engine/src/main/java/org/terasology/world/block/BlockBuilderHelper.java

```diff
@@ -21,23 +21,101 @@
 import org.terasology.world.block.loader.SectionDefinitionData;
 import org.terasology.world.block.shapes.BlockShape;
 
+/**
+ * A helper class for building and registering blocks with specific properties.
+ * Intended for use with Block Family related code.
+ * <p>
+ * The core features of this helper is to convert from definition data, into a Block
+ * and to register this block into the engine for use.
+ */
 public interface BlockBuilderHelper {
 
+    /**
+     * {@inheritDoc}
```

Comment:
What does this do? We are in an interface, where should we inherit from?

Reply:
Nothing useful, I removed it.